cmd/api/src/model/model.go

```diff
@@ -97,4 +97,5 @@ type SearchResult struct {
 	Type              string `json:"type"`
 	Name              string `json:"name"`
 	DistinguishedName string `json:"distinguishedname"`
+	SystemTags        string `json:"system_tags"`
```

Comment:
I already approved this MR but actually I think we need to update the API docs with this change